metals/src/main/scala/scala/meta/internal/metals/UserConfiguration.scala

```diff
@@ -39,6 +39,7 @@ case class UserConfiguration(
     bloopSbtAlreadyInstalled: Boolean = false,
     bloopVersion: Option[String] = None,
     bloopJvmProperties: Option[List[String]] = None,
+    bloopJavaHome: Option[String] = None,
```

Comment:
I would just use the same java Home as the Metals one it will be a bit confusing for the users otherwise.

Reply:
Ok, so I would remove it!